src/tests/calculate.test.js

```diff
@@ -0,0 +1,19 @@
+import calculate from '../logic/calculate';
+
+describe('calculate', () => {
+  test('calculate', () => {
+    const obj = {
+      total: 3,
+      next: 4,
+      operation: '+',
+    };
+    const obj2 = {
+      total: null,
+      next: null,
+      operation: null,
+    };
+    const answer = calculate(obj, 'AC');
+    expect(JSON.stringify(answer)).toEqual(JSON.stringify(obj2));
+    expect(calculate({ next: '5' }, 'AC')).toHaveProperty('total', null);
+  });
+});
```

Comment:
- [ ] You did a good job working on this milestone! 💯 However, according to the requirements:

![image](https://user-images.githubusercontent.com/77069011/236084215-6eb1c8b1-2584-48ae-8653-516413659c6d.png)

Please make sure to test all functionalities and scenarios of the `calculate.js` file, at the moment you are just testing one! 🤓 🔧

Reply:
well resolved! ✔️